docs/source/deployguide/ca-deploy-topology.md

```diff
@@ -0,0 +1,58 @@
+
+You may already be familiar with the Fabric CA [User's Guide](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/users-guide.html) and the [Operations Guide](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/operations_guide.html). This topic is intended to inform your decisions before deploying a CA and provide guidance on how to configure the CA parameters based on those decisions. You may still need to reference those topics when you make your decisions.
+
+Recall that a Fabric CA performs the following functions on a blockchain network:
+- Registration of identities, or connect to LDAP as the user registry.
+- Issuance of Enrollment Certificates (ECerts). Enrollment is a process whereby the Fabric CA issues a certificate key-pair, comprised of a signing certificate and a private key that forms the identity. The private and public key are first generated locally by the Fabric CA client, and then the public key is sent to the CA which returns an encoded certificate, the signing certificate.
+- Certificate renewal and revocation.
+
+You have the opportunity to customize the behavior of these functions. The first time the CA is started, it looks for a [fabric-ca-server-config.yaml
+file](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/serverconfig.html) which contains the CA configuration parameters. If the file is not there, a default one is created for you. Before you deploy your CA, this topic provides guidance around the parameters in that file and the decisions you need to make in order to customize the CA according to your use case.
+
+## What CA topology will you use on your network?
+
+The topology of CAs on your network can vary depending on how many organizations participate on the network and how you prefer to administer your CAs.
+
+### How many CAs are required?
+
+Before configuring a CA, you need to understand how many CAs are required for your network. If you read the Deployment Process Overview, you'll recall that it is recommended that you deploy two CAs per organization, an enrollment CA and a TLS CA. TLS communications are required for any production network to secure communications between nodes in the organization. Thus, it is the TLS CA that issues those TLS certificates. The enrollment CA on the other hand is used to generate organization and node identities. Also, because this is a _distributed ledger_, the ordering service should not be part of the same organization as the peers, so you will need separate organizations (and therefore CAs) for your peer organizations and ordering service organization. And if multiple organizations contribute nodes to an ordering service, each ordering node would have its own organization CA.
```

Comment:
“Also, because this is a distributed ledger, the ordering service should not be part of the same organization as the peers, so you will need separate organizations (and therefore CAs) for your peer organizations and ordering service organization. “

This strikes me an insufficient explanation. isn't everything in Fabric is distributed? Is there a more specific reason? (such as channel management or the risk profile of the channel)

Reply:
Good suggestion. I changed it to this:

Also, because this is a _distributed ledger_, the ordering service should not be part of the same organization as the peers, so you will need separate organizations (and therefore CAs) for your peer organizations and ordering service organization. When multiple organizations contribute nodes to an ordering service, each ordering node would have its own organization CA. All of this separation is crucial for distributed management of the ordering service and channels and defeats the ability of a bad actor to disrupt the network.

Reply:
👍